scripts/dmc_reset.py

```diff
@@ -127,18 +130,18 @@ def parse_args():
 
 def reset_dmc(args):
```

Comment:
Would be kind of nice to keep `reset_dmc(args)`, I think, rather than rework that function to use dicts everywhere. I get that a dict is convenient to compose arguments on the fly in a test, but there is an alternative that I'll suggest in that file.